docs/siem/detections/api/rules-api-create.asciidoc

```diff
@@ -360,7 +450,79 @@ When it runs next, at 16:00, it will analyze data from 14:50 until 16:00.
 
 *Example 2*
 
-{ml-cap} rule that creates signals, and sends Slack notifications, when the
+Threshold rule that detects multiple failed login attempts to a Windows host
+from the same external source IP address, and maps the `severity` value to
+custom source event fields:
+
+[source,console]
+--------------------------------------------------
+POST api/detection_engine/rules
+{
+  "description": "Detects when there are 20 or more failed login attempts from the same IP address with a 2 minute time frame.",
+  "enabled": true,
+  "exceptions_list": [ <1>
+    {
+      "id": "int-ips",
+      "namespace_type": "single",
+      "type": "detection"
+    }
+  ],
+  "from": "now-180s",
+  "index": [
+    "winlogbeat-*"
+  ],
+  "interval": "2m",
+  "name": "Liverpool Windows server prml-19",
+  "query": "host.name:prml-19 and event.category:authentication and event.outcome:failure",
+  "risk_score": 30,
+  "rule_id": "liv-win-ser-logins",
+  "severity": "low",
+  "severity_mapping": [ <2>
```

Comment:
I'll need to verify the compatibility constraints of using the severity/risk_score mappings with Threshold Rules. I believe that for this mapping to be successful, any `field` used within the `mapping` will also have to be referenced in the rule's query so that there exists a unique value per bucket that can then be used for the mapping (otherwise it will fall back to the default value since the field will be missing).

Will keep you posted on this one 🙂

Reply:
Ahh ok. If so, we'll document the constraint in the UI section, and I'll update the example.